### PR TITLE
readme, license, notice, disclaimer

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**

### CODE REVIEW

File: DISCLAIMER [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+completeness or stability of the code, it does indicate that the
+project has yet to be fully endorsed by the ASF.
+
+Some of the incubating project's releases may not be fully compliant
+with ASF policy. For example, releases may have incomplete or
+un-reviewed licensing conditions. What follows is a list of known
+issues the project is currently aware of (note that this list, by
+definition, is likely to be incomplete): 
+
+ * The LICENSE file is incomplete and will be fixed with the next release.
```
Comment (leerho) [inline DISCLAIMER]: I would change this to "The LICENSE file may be incomplete..."

File: DISCLAIMER [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Apache DataSketches (incubating) is an effort undergoing incubation
```
Comment (leerho) [inline DISCLAIMER]: The name of this file should be DISCLAIMER-WIP. (Ref: http://incubator.apache.org/policy/incubation.html#Releases).

File: DISCLAIMER (hunk 1 above)
Comment (AlexanderSaydakov) [reply DISCLAIMER]: changed

File: DISCLAIMER (hunk 2 above)
Comment (AlexanderSaydakov) [reply DISCLAIMER]: renamed

**Pull request merged**